.github/workflows/backport-pr.yml

```
@@ -0,0 +1,63 @@
+        with:
+          route: GET /repos/longhorn/longhorn/issues/${{ env.ORIGINAL_ISSUE_NUMBER }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: URL encode the original issue title
+        if: env.BACKPORT == 'true'
+        run: echo "ORIGINAL_ISSUE_TITLE=$(node -e 'console.log(encodeURIComponent("${{ fromJson(steps.original-issue.outputs.data).title  }}"))')" >> $GITHUB_ENV
+
+      - name: Find corresponding backport issue number
+        if: env.BACKPORT == 'true'
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:56: Double quote to prevent globbing and word splitting

(shellcheck)